### DISCUSSION TITLE
AMD Ryzen 9 9950X 16-Core Processor

### DISCUSSION THREAD

**Discussion by pgarba** [isc30/ryzen-gpu-passthrough-proxmox#72] [General]
I can confirm that it works on the AMD Ryzen 9 9950X 16-Core Processor!

Attached the GPU Rom and I used the `AMDGopDriver_8945.rom` for audio. 

[vbios_9950.bin.zip](https://github.com/user-attachments/files/17327455/vbios_9950.bin.zip)

**Comment by realies** [comment]
@pgarba, is the 9950x iGPU expected to work without a display connected to it when forced in bios?
Still getting error 43 after loading this rom.

**Comment by billythekidz** [comment]
I confirmed that https://github.com/user-attachments/files/17327455/vbios_9950.bin.zip worked for my iGPU 
Thank @pgarba  👍
